content/400-reference/200-api-reference/200-command-reference.mdx

```diff
@@ -571,229 +568,148 @@ Push the schema with a custom schema location:
-
-| Argument  | Required | Description                                                                             | Default |
-| --------- | -------- | --------------------------------------------------------------------------------------- | ------- |
-| decrement | No       | Specifies the number of backwards migrations to apply.                                  | 1       |
-| name      | No       | Specifies where to migrate back to using the name of the final migration to apply.      |
-| timestamp | No       | Specifies where to migrate back to using the timestamp of the final migration to apply. |
-
-#### Options
-
-Additionally, the following options modify the behavior of the `migrate down` command:
+| Argument       | Required | Description                                                                                                                                       | Default                                          |
+| :------------- | :------- | :------------------------------------------------------------------------------------------------------------------------------------------------ | :----------------------------------------------- |
+| `--applied` | No*       | Record a specific migration as applied - for example `--applied "20201231000000_add_users_table"` |  |
+| `--rolledback` | No*       | Record a specific migration as rolled back - for example `--rolledback "20201231000000_add_users_table"`. | `./schema.prisma`<br /> `./prisma/schema.prisma` |
+| `--schema`    | No       | Specifies the path to the desired schema.prisma file to be processed instead of the default path. Both absolute and relative paths are supported. | `./schema.prisma`<br /> `./prisma/schema.prisma` |
 
-| Option            | Required | Description                                                                                                                                         | Default                                     |
-| ----------------- | -------- | --------------------------------------------------------------------------------------------------------------------------------------------------- | ------------------------------------------- |
-| `--experimental`  | Yes      | Enables use of Experimental commands                                                                                                                |                                             |
-| `-p`, `--preview` | No       | Preview the migration that would be created without writing any changes to the filesystem.                                                          |                                             |
-| `--schema`        | No       | Specifies the path to the desired `schema.prisma` file to be processed instead of the default path. Both absolute and relative paths are supported. | `./schema.prisma`, `./prisma/schema.prisma` |
+You must specify either `--rolledback` _or_ `--applied`.
```

Comment:
```suggestion
You must specify either `--rolled-back` _or_ `--applied`.
```
Also could you check everywhere that it's `--rolled-back` and not `--rolledback`?